stylesheets/style4.css

```diff
@@ -0,0 +1,365 @@
+    font-weight: 300;
+    margin: 0 auto;
+    text-transform: uppercase;
+    text-align: center;
+    position: absolute;
+    z-index: 9;
+    
+}
+.headerWrap h1{
+    font-size: 5rem;
+    letter-spacing: 15px;
+    top: 300px;
+    left: 350px;
+    
+}
+
+.col1 h1{
+    font-size: 2.5rem;
+    top: 450px;
+    left: 550px;
+    word-spacing: 10px;
+}
+
+.landing .col1 h3{
+    font-size: 1rem;
+    top: 750px;
+    left: 550px;
+    letter-spacing: 1px;
+}
+
+h2, h3, h4, h5, h6 {
+    font-family: 'Cookie', cursive,'Sansita Swashed', cursive;
+    font-weight: 300;
+    letter-spacing: 1px;
+    margin: 0;
+}
+p {
+    font-family: 'Lato', sans-serif;
+    font-weight: 300;
+    margin: 0;
+}
+
+a {
+    color: inherit;
+    text-decoration: none;
+}
+
+a:hover {
+    text-decoration: underline;
+}
+
+.landingWrap img{
+    top: 180px;
+    left: 35%;
+    max-width: 580px;
+    position: relative;
+}
+
+.landing{
+    background-color: #f8c573;
+    height: 100vh;
+}
+
+button{
+    background-color: #82450f;
+    width: 250px;
+    height: 40px;
+    border: transparent;
+    font-weight: bold;
+    color: white;
+    font-size: 15px;
+    border-radius: 5%;
+    transition: 0.3s;
+    cursor: pointer;
+    margin-top: 800px;
+    margin-left: 820px;
+    position: absolute;
+}
+/*
+.sectionWrap {
+    padding: 75px 0;
+    overflow: hidden;
+    position: relative;
+}
+
+.sectionWrap .col1 {
+    max-width: 1920px;
+    margin: 0 auto;
+}
+
+.sectionWrap .col1 h2 {
+    font-size: 180%;
+    text-transform: uppercase;
+    text-align: center;
+}
+
+.sectionWrap .col1 p {
+    margin: 25px 0;
+    letter-spacing: 1px;
+    text-align: center;
+}
+
+.sectionWrap .col2 {
+    max-width: 25%;
+    margin: 0 auto;
+}
+
+.sectionWrap .col2{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+}
+
+.card1,.card2,.card3,.card4{
+    display: inline;
+}
+
+.sectionWrap .col2 .card1,
+.sectionWrap .col2 .card2,
+.sectionWrap .col2 .card3,
+.sectionWrap .col2 .card4 {
+    width: 100%;
+    color: var(--font-dark);
+    background-color: var(--heading);
+    box-shadow: inset 0px 0px 15px rgba(0, 87, 255, 0.5);
+    border-radius: 5px;
+    padding: 25px;
+    overflow: hidden;
+    margin: 50px 0;
+}
+
+.sectionWrap .col2 .card1 img,
+.sectionWrap .col2 .card2 img,
+.sectionWrap .col2 .card3 img,
+.sectionWrap .col2 .card4 img {
+    width: 50%;
+    padding: 20px;
+    float: right;
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.card1>h4,.card2>h4,.card3>h4,.card4>h4{
+    font-family: 'Maven Pro', sans-serif;
+    text-transform: uppercase;
+    text-align: center;
+}
+.card1>img,.card2>img,.card3>img,.card4>img{
+    max-height: 30%;
+    margin: 0 40%;
+}
+
+.footerWrap .col2 svg,
+.footerWrap .col2 svg,
+.footerWrap .col2 svg,
+.footerWrap .col2 svg{
+    height: 30px;
+    fill: var(--font);
+}*/
```

Comment:
These are not needed, you can remove them

Reply:
All in all, this stylesheet @himanshubobade is fantastic :100: 

Please make it responsive and then we will get it merged.

Reply:
@faisalAkhtar yes will do soon